pkg/operator/ceph/operator.go

```diff
@@ -101,6 +107,22 @@ func (o *Operator) cleanup(stopCh chan struct{}) {
 	o.clusterController.StopWatch()
 }
 
+func (o *Operator) updateOperatorLogs() error {
+	logLevel, err := k8sutil.GetOperatorSetting(o.context.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_LOG_LEVEL", "INFO")
+	if err != nil {
+		logger.Warningf("failed to load ROOK_LOG_LEVEL. Defaulting to INFO. %v", err)
+		logLevel = "INFO"
+	}
+
+	log, err := capnslog.ParseLevel(logLevel)
+	if err != nil {
+		logger.Warningf("failed to set log level %q. %+v", log, err)
```

Comment:
need to return or else `log` is invalid when used below

Reply:
added return